examples/src/main/scala/TracingExample.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import cats.effect.IO
+import cats.effect.IOApp
+import cats.effect.MonadCancelThrow
+import cats.effect.std.Console
+import cats.syntax.all._
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk
+import org.typelevel.otel4s.java.OtelJava
+import org.typelevel.otel4s.trace.Tracer
+
+trait Work[F[_]] {
+  def doWork: F[Unit]
+}
+
+object Work {
+  def apply[F[_]: MonadCancelThrow: Tracer: Console]: Work[F] =
+    new Work[F] {
+      def doWork: F[Unit] =
+        implicitly[Tracer[F]].span("Work.DoWork").use { span =>
+          span.addEvent("Starting the work.") *>
+            doWorkInternal *>
+            span.addEvent("Finished working.")
+        }
+
+      def doWorkInternal =
+        Console[F].println("Doin' work")
+    }
+}
+
+class TracingExample extends IOApp.Simple {
+  def getTracer: IO[Tracer[IO]] =
+    for {
+      otel4j <- IO(
+        AutoConfiguredOpenTelemetrySdk.builder.build().getOpenTelemetrySdk
+      )
+      otel4s <- OtelJava.forSync(otel4j)
+      tracerProvider = otel4s.tracerProvider
+      tracer <- tracerProvider.tracer("Example").get
+    } yield tracer
```

Comment:
I'd say this might make a worthwhile helper, but we'd have to take on a dependency on an alpha otel-java module.

Reply:
As was mentioned in the [Discord](https://discord.com/channels/632277896739946517/918373380003082250/1065662291548311612), it would also be nice to have a set of helper (builder?) methods to make a valid SDK with configured propagators, id generators, etc. 

The following example was mentioned: 
https://github.com/Dwolla/dwolla-otel-natchez/blob/c65b4953c0dc23766c061ab6d2cb8549f79066e6/core/src/main/scala/com/dwolla/tracing/OpenTelemetryAtDwolla.scala#L40-L46

Reply:
I think all of that can be configured with environment variables using the auto-configured SDK, which seems like a good way.

This still needs to handle that null that it can return, though.

Reply:
I'd definitely lean on the env variables that are provided by Otel as well.  Non-custom configuration is really key to making things like this usable for folk / orgs.